### PR TITLE
Overmind over Hivemind to allow developer to run byebug in Rails

### PR DESCRIPTION
**Pull request by user512**
To allow developer to run debugger in convene-web.

### CODE REVIEW

**Comment by user512** [comment]
Blocked by https://github.com/zinc-collective/convene/pull/225, rebase `development` after #225 is merged in.

File: .github/workflows/test-convene-web.yml
Comment (zspencer) [inline .github/workflows/test-convene-web.yml:42]: Do we need this since `brew install overmind` works in CI?

**Review by zspencer — APPROVED** [review]
Thanks Tom!

File: .github/workflows/test-convene-web.yml
Comment (user512) [reply .github/workflows/test-convene-web.yml:42]: Right! Let me get rid of this.

**Pull request merged**
